src/test/resources/fake-agency-overlapping-trips/frequencies.txt

```diff
@@ -0,0 +1,3 @@
+trip_id,start_time,end_time,headway_secs,exact_times
+freq-01,09:00:00,10:00:00,600,0
+freq-01,09:30:00,10:30:00,900,0
```

Comment:
Can you add some more frequencies to this feed? Specifically, I think it'd make for a great test if we had all of the conditions recommended in https://github.com/conveyal/gtfs-lib/pull/236/files#r287460184. Also, it'd be great to add some valid frequency entries to make sure there are no false positives. ie `Freq A` is entirely before `Freq B` and `Freq A` is entirely after `Freq B`. Bonus points for adding even more edge cases like Freq A ends exactly when Freq B starts. Super bonus points for adding frequency entries that have interactions with daylight savings time transitions.